spice/deps/helper_cb.py

```diff
@@ -27,12 +27,12 @@
 import os
 import logging
 import argparse
+from PIL import Image, ImageDraw
```

Comment:
>>> from PIL import Image
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named PIL

This package is provided by RPM: python-pillow
It is necessary to add this RPM name to
tp-spice/requirements-travis.txt
tp-spice/requirements.txt

Yes, it seems that  Image, ImageDraw are modules. (Which violates python naming convention).

Okay, than: https://www.python.org/dev/peps/pep-0008/#imports is still important.